requirements_dev.txt

```diff
@@ -0,0 +1,3 @@
+pyfftw>=0.12.0
```

Comment:
Please remove this file unless you have a strong opinion. `pytest` could go into `tests/requirements.txt`, but I can refactor that (and likely the CI) after this PR.

Reply:
Yes, that would make sense.